spec/acceptance/hieradata/common.yaml

```diff
@@ -0,0 +1,4 @@
+---
+jenkins::service_override:
+  StartLimitIntervalSec: '0'
+  StartLimitBurst: '0'
```

Comment:
Someone was so smart to set
```
StartLimitBurst=5
StartLimitIntervalSec=5m
```
as the default in the `jenkins.service` file -- we restart it a tad more often during tests, and well, at some point systemd refuses to restart further.

Reply:
Wouldn't hurt to have that in the commit message or comment, but here in the PR is a good start